templates/new/batch_main.py.txt

```diff
@@ -12,6 +12,8 @@ def main():
 
 def execute_assembler(mode):
     assembler = Assembler("Default project", ValidateData(), Main())
+    assembler.load_config("{project_name}")
```

Comment:
`load_config` should not need to be called by the generated code and really should be a part of the Assembler constructor. Please make that modification.